roles/test-utils/sv1-mining-device/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "sv1-mining-device"
+name = "sv1_mining_device"
```

Comment:
It makes sense to keep this consistent with other crates in similar roles. I'm not sure if we've published this crate, but if we have, we should deprecate the previous version.

Reply:
it has not yet been published to `crates.io`, so it's still not too late to change things

Reply:
+1 on consistency for naming patterns

Reply:
whats the consistency? I don't think we can use `-` in library names, that's why I changed this..

Reply:
`_` instead of `-` is in the right direction

I would switch `sv1` to suffix instead of prefix: `mining_device_sv1`

that's what we have on other roles, e.g.: `pool_sv2`, `mining_proxy_sv2`